lib/Doctrine/ORM/Mapping/Driver/YamlDriver.php

```diff
@@ -922,6 +922,8 @@ private function cacheToArray(array $cacheMapping): array
             $usage = (int) constant('Doctrine\ORM\Mapping\ClassMetadata::CACHE_USAGE_' . $usage);
         }
 
+        assert($usage !== '0' && $usage !== '');
```

Comment:
Can we be sure? I think we should either map all falsy values to `null` or make sure falsy values other than null also trigger an `InvalidArgumentException`.

Reply:
I think you're right, but let's maybe not do this in a patch release?

Reply:
Is a suddenly failing `assert()` better in a patch release? 😬

Reply:
Provided assertions are not enabled in production, I'd say yes.

Reply:
Not a doc block (as the title suggests) though, but in comparison it screams at dev runtime when something's wrong. :-)